azure-pipelines-build.yml

```diff
@@ -255,6 +257,44 @@ extends:
           - publish: $(System.DefaultWorkingDirectory)/functions.zip
             artifact: function-handle-subscriptions
             displayName: Publish Functions Artifact
+      - name: Build Function - Deadline Submissions
+        condition: >-
+          and(
+            succeeded(),
+            or(
+              and(
+                eq(variables['Build.Reason'], 'Manual'),
+                eq(${{ parameters.buildDeadlineSubmissionsFunction }}, 'true')
+              ),
+              and(
+                ne(variables['Build.Reason'], 'Manual'),
+                or(
+                  eq(variables['Build.SourceBranchName'], 'main'),
+                  eq(dependencies.run_linting_and_tests.outputs['check_files.functions-deadline-submissions'], 'true')
+                )
+              )
+            )
+          )
+        dependsOn:
+          - Run Linting & Tests
+        steps:
+          - template: ../steps/node_script.yml
+            parameters:
+              nodeVersion: 16
+              script: npm install $(npm pack $(Build.Repository.LocalPath)/packages/event-client) $(npm pack $(Build.Repository.LocalPath)/packages/platform) --only=prod --workspace=@pins/functions-deadline-submissions
```

Comment:
I think you need to add your other `@pins` dependencies here as extra `npm pack` lines, e.g. `blob-storage-client`